modules/virt-uploading-local-disk-image-dv.adoc

```diff
@@ -64,6 +64,8 @@ $ virtctl image-upload dv <datavolume_name> \ <1>
 ====
 * If you do not want to create a new data volume, omit the `--size` parameter and
 include the `--no-create` flag.
+* When uploading a disk image to a PVC, the PVC size must be larger than the size of the uncompressed virtual disk.
+====
```

Comment:
@bgaydosrh I think you need to remove this extra `====` here. That might be the source of error.

Reply:
Thanks. I didn't see the third note :-(